### PR TITLE
UI Tests For Request Workflows

### LINKED ISSUE #90: Create tests for the request workflow
We need to add cypress tests for the following:

1. Request acceptance
1. Request release
1. Request completion
1. Add admin comment on a request
1. Add private comment on a request

### PR DESCRIPTION
**Pull request by prescottprue**
### Description
* Adds tests for the following cases (as described by #90):
  1. Request acceptance
  1. Request release
  1. Request completion
  1. Admin can add public comment on a request
  1. Add private comment on a request
* Adds loading of test variables from .env files

### Relevant Tickets (Please add `closes`, `refs`, etc)

- closes #90

### CODE REVIEW

File: src/routes/Request/components/RequestActions/RequestActions.js [hunk 1]
```diff
@@ -23,7 +24,10 @@ function RequestActions({ requestPublicSnapshot }) {
   const [showAcceptanceWorkflow, setShowAcceptanceWorkflow] = useState(false);
   const [showReleaseWorkflow, setShowReleaseWorkflow] = useState(false);
   const [showCompletionWorkflow, setShowCompletionWorkflow] = useState(false);
-
+  const firestore = useFirestore();
+  const userProfileSnap = useFirestoreDoc(
+    firestore.doc(`${USERS_COLLECTION}/${user.uid}`),
```
Comment (thebitguru) [inline src/routes/Request/components/RequestActions/RequestActions.js]: Unauthenticated users can see requests too, so, this is currently failing.

**Review by thebitguru — CHANGES_REQUESTED** [review]
This looks good, but we are missing the public comment tests.

Also, what's the thought on clearing our the database or specific records before/after the run?  This way it's easier to re-run the cypress tests without having to restart the emulator.

File: src/routes/Request/components/RequestActions/RequestActions.js (hunk 1 above)
Comment (prescottprue) [reply src/routes/Request/components/RequestActions/RequestActions.js]: Oh shoot, ok - I was trying to add in the support for if you are system-admin, I can remove

Comment (prescottprue) [reply src/routes/Request/components/RequestActions/RequestActions.js]: Removed

File: cypress/integration/RequestPage.spec.js [hunk 2]
```diff
@@ -3,26 +3,332 @@ import { createSelector } from '../utils';
...
+        phone,
+      });
+      // Authenticate user
+      cy.login();
+      // Visit request page
+      cy.visit(`/requests/${requestId}`);
+    });
+
+    after(() => {
+      // Set current user to super-admin role
```
Comment (thebitguru) [inline cypress/integration/RequestPage.spec.js:46]: Is this comment correct?  This implies that `role=null` is system admin, no?

**Review by thebitguru — APPROVED** [review]

**Pull request merged**